docs/developer-guide/table-functions.rst

```diff
@@ -0,0 +1,90 @@
+.. _table-functions:
+
+
+Using Table Functions With KSQL

```

Comment:
this should probably be inside `udf.rst` where we have UDF/UDAF documentation (including all annotation documentation)

Reply:
done